packages/cisco_secure_email_gateway/data_stream/log/elasticsearch/ingest_pipeline/pipeline_text_mail_logs.yml

```
@@ -6,33 +6,89 @@ processors:
+        - '^%{GREEDYDATA:cisco_secure_email_gateway.log.description}: Unable to send System/Warning %{DATA:event.kind} to %{DATA:email.to.address} with subject \"%{GREEDYDATA:email.subject}\".$'
+        - '^Internal %{DATA:network.protocol} system successfully sent a message to %{DATA:email.to.address} with subject %{GREEDYDATA:email.subject}.$'
+        - '^Internal %{DATA:network.protocol} giving up on message to %{DATA:email.to.address} with subject %{GREEDYDATA:email.subject}.$'
+        - '^Internal %{DATA:network.protocol} Error: %{GREEDYDATA:cisco_secure_email_gateway.log.description} to host %{IP:destination.ip}:%{NUMBER:destination.port:long} for recipient %{DATA:email.to.address}: %{GREEDYDATA:cisco_secure_email_gateway.log.subject}$'
+        - '^MID %{NUMBER:email.message_id} RID \[%{DATA:cisco_secure_email_gateway.log.recipient_id}\] Response %{GREEDYDATA:cisco_secure_email_gateway.log.response}$'
+        - '^MID %{NUMBER:email.message_id} Subject \"%{GREEDYDATA:cisco_secure_email_gateway.log.subject}\"$'
+        - '^MID %{NUMBER:email.message_id} %{GREEDYDATA:cisco_secure_email_gateway.log.message_status} for delivery$'
+        - '^Message %{GREEDYDATA:cisco_secure_email_gateway.log.message_status} MID %{NUMBER:email.message_id} done$'
+        - '^MID %{NUMBER:email.message_id} interim verdict using engine\: %{GREEDYDATA:cisco_secure_email_gateway.log.engine}$'
+        - '^MID %{NUMBER:email.message_id} interim AV verdict using %{GREEDYDATA:cisco_secure_email_gateway.log.engine}$'
+        - '^MID %{NUMBER:email.message_id} using engine\: %{GREEDYDATA:cisco_secure_email_gateway.log.engine}$'
+        - '^MID %{NUMBER:email.message_id} %{GREEDYDATA:cisco_secure_email_gateway.log.subject}\: verdict %{GREEDYDATA:cisco_secure_email_gateway.log.verdict_scale}$'
+        - '^MID %{NUMBER:email.message_id} Message-ID ''\<%{GREEDYDATA:cisco_secure_email_gateway.log.email}\>\''$'
+        - '^MID %{NUMBER:email.message_id} %{DATA:network.protocol}\: Verification %{GREEDYDATA:cisco_secure_email_gateway.log.verified}$'
+        - '^MID %{NUMBER:email.message_id} %{DATA:network.protocol}\: mailfrom identity %{DATA:email.from.address} %{GREEDYDATA:cisco_secure_email_gateway.log.verified} \(v\=spf1\)$'
+        - '^MID %{NUMBER:email.message_id} matched all recipients for per-recipient policy %{WORD:cisco_secure_email_gateway.log.policy} in the %{DATA:email.direction} table$'
+        - '^MID %{NUMBER:email.message_id} SDR\: Tracker Header \: %{GREEDYDATA:cisco_secure_email_gateway.log.email_tracker_header}$'
+        - '^MID %{NUMBER:email.message_id} SDR\: Domains for which SDR is requested\: reverse DNS host\: %{DATA:dns.question.name}\, helo\: %{DATA:cisco_secure_email_gateway.log.helo}\, env-from\: %{DATA:cisco_secure_email_gateway.log.env}\, header-from\: %{DATA:email.from.address}\, reply-to\: %{DATA:email.to.address}$'
```

```suggestion
        - '^MID %{NUMBER:email.message_id} SDR: Domains for which SDR is requested: reverse DNS host: %{DATA:dns.question.name}, helo: %{DATA:cisco_secure_email_gateway.log.helo}, env-from: %{DATA:cisco_secure_email_gateway.log.env}, header-from: %{DATA:email.from.address}, reply-to: %{DATA:email.to.address}$'
```
? (similar elsewhere)